u7-components/src/views/BookCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <router-link
+    :to="{ name: 'BookDetails', params: { isbn13: book.isbn13 } }"
```

Comment:
อ่าฮะ! สิ่งนี้จะไม่ทำงาน เนื่องจากคุณตั้งชื่อพารามิเตอร์ `bookId` ในเราเตอร์ของคุณ แต่ที่นี่คุณตั้งค่าของพารามิเตอร์อื่น `isbn13' เปลี่ยนเราเตอร์หรือตั้งค่า `bookId` // Ah ha! This will not work, because you named the parameter `bookId` in your router, but here you set the value of a different parameter, `isbn13`. Either change the router, or set `bookId`:
```ts
:to="{ name: 'BookDetails', params: { bookId: book.isbn13 } }"
```

Reply:
หนังสือมีเอกลักษณ์เฉพาะตัวในการมีชื่อเฉพาะสำหรับบัตรประจำตัว อีกตัวอย่างหนึ่งคือ `sku`
Books are fairly unique in having a specific name for their ID. Another common one is `sku`.